Softeq.XToolkit.Connectivity/ConnectivityService.cs

```diff
@@ -32,17 +27,8 @@ public ConnectivityService()
 
         public void Dispose()
```

Comment:
You need to add an `IDisposable` interface

Reply:
It's implementing `IDisposable` implicitly because `IConnectivityService` is derived from `IDisposable`